src/rust/mangastream/sources/kanzenin/res/source.json

```diff
@@ -0,0 +1,15 @@
+{
+	"info": {
+		"id": "id.kanzenin",
+		"lang": "id",
+		"name": "Kanzenin",
+		"version": 1,
+		"url": "https://kanzenin.xyz/",
+		"nsfw": 2
+	},
+	"listings": [
+		{ "name": "Popular" },
+		{ "name": "Latest" },
+        { "name": "New" }
```

Comment:
Make sure you format JSON files so they're consistent (use tabs instead of spaces).
```sh
find . -type f -name '*.json' -exec sh -c 'jq --tab . {} | sponge {}' \;
```

Reply:
fixed!

Reply:
I also add 1 more sources for Indonesian Language.